src/utils/mutations/__tests__/LogUserRevenueMutation.test.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jest */
```

Comment:
You can remove this eslint line. Our config in this repo will assume `*.test.js` files are a Jest env.